sql/parser/sql.y

```diff
@@ -1003,19 +1003,17 @@ set_reset_clause:
   SET set_rest {}
 
 show_stmt:
-  SHOW var_name
-  {
-    // It would be cleaner if we could have "SHOW DATABASES" and "SHOW
-    // TABLES" rules, but unfortunately DATABASES and TABLES are
-    // unreserved keywords and thus valid variable names and such
-    // rules would cause reduce/reduce conflicts.
-    if strings.EqualFold($2.String(), `"DATABASES"`) {
-      $$ = &ShowDatabases{}
-    } else if strings.EqualFold($2.String(), `"TABLES"`) {
-      $$ = &ShowTables{}
-    } else {
-      $$ = nil
```

Comment:
Hmm, you've removed the unimplemented support for `SHOW var_name`. That's what fixed the reduce/reduce conflicts. Not sure if we'll need to re-add this or not.